apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/PromptConfiguration/index.tsx

```diff
@@ -35,16 +39,19 @@ export function PromptConfiguration({
         <div className='flex flex-col gap-6 p-2'>
           <GenerationSettings
             config={config}
+            canUseSubagents={canUseSubagents}
             setConfig={setConfig}
             disabled={disabled}
           />
           <BehaviourSettings
             config={config}
+            canUseSubagents={canUseSubagents}
             setConfig={setConfig}
             disabled={disabled}
           />
           <LimitSettings
             config={config}
+            canUseSubagents={canUseSubagents}
```

Comment:
this was perfect use case for a little context instead of prop drilling to n components